<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SingUp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/image/window_icon.png</normaloff>:/image/image/window_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#signup{

border-image:  url(:/image/Background03.png) 0 0 0 0 stretch stretch;}

</string>
  </property>
  <widget class="QGroupBox" name="groupBox_register">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>401</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_register{font: 19pt &quot;Segoe UI&quot;;
color:White;

border-radius: 15px;

background-color: #841ebc;
border: 1px solid #a23fdf;}</string>
   </property>
   <property name="title">
    <string> Please Register</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>281</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_register_password">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_register_password">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>281</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_register_email">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_register_email">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>281</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_register_phone">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Phone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_register_phone">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_register">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: 2px solid #ffffff;
font: 10pt &quot;Segoe UI&quot;;
  border-radius: 15px;
color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>281</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_register_username">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;UserName&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_register_username">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
border: 2px solid  white;
 border-radius: 5px;
background-color: transparent;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  border: 1px solid #ff99cc;
  border-radius: 10px;
color: #ff99cc;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>710</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #000000;
background-color:#000000;
border-radius: 15px;
color: white;
font: 700 10pt &quot;Segoe UI&quot;;
	</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
